tests/test_dataset_wrapper.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+import pandas as pd
+import pytest
+from sklearn.model_selection import train_test_split
+
+from datasets.dataset_wrapper import DatasetWrapper
+
+wine_data_path = "../datasets/winequality.csv"
```

Comment:
Hardcoding the path here is not a great idea as it makes assumptions about which working dir the tests are run from.

In fact I noticed this because I tried running the tests from the root dir :).

A way to get around that is to use
```python
from pathlib import Path
Path(__file__)
```
to get the path of this module (`test_dataset_wrapper.py`), and set paths relative to that.

Reply:
Took your suggestion and now everything is in reference to Path! Thank you for the catch.